browser-test/src/support/index.ts

```diff
@@ -180,6 +180,14 @@ export const seedCanonicalQuestions = async (page: Page) => {
   await page.click('#canonical-questions')
 }
 
+export const enableFeatureFlag = async (page: Page, flag: string) => {
+  await page.goto(BASE_URL + '/dev/feature/$flag/enable')
+}
+
+export const disableFeatureFlag = async (page: Page, flag: string) => {
+  await page.goto(BASE_URL + '/dev/feature/$flag/disable')
+}
```

Comment:
The above interpolation leads me to believe that the calls to enableFeatureFlag / disableFeatureFlag are a no-op.

Given the current implementation where probers would entirely skip the tests, do we actually need to have the enable/disable as part of this PR?

At a high level, I think something that might be missing is once again disabling status tracking in the relevant `.conf` file. We may want to add a new one specifically for browser tests that inherits from whatever is there right now and disables the feature. That would hopefully make it explicit around whether the feature flag forcing mechanism was working or not.

Reply:
You were correct, I fixed that bug.

We also need to disable the .conf value which is a few more steps.  I made #3237 for that